### PR TITLE
Updated creating organization so the 'complete' button also updates the information

### PR DESCRIPTION
**Pull request by ConnorCartwright**
Hey guys,

Whilst working on [#448](https://github.com/education/classroom/issues/448) I noticed that when you create an organization, if you update the organization name in the screen:

And click **_Continue_** rather than _**Save Changes**_ the organization name does not get updated.

I wasn't exactly sure where to put the code to fix this or if there's a better way so I'm open to suggestions. 

Currently I have removed the submit function from the save changes button and added a class to both buttons that I look for in the `update_organization.coffee` file and submit the form on click.

Cheers,
Connor

### CODE REVIEW

**Comment by dinever** [comment]
Hi @ConnorCartwright 

> And click Continue rather than Save Changes the organization name does not get updated.

Seems that this is an intended behavior.

Actually, if clicking `Continue` also updates the organization, then the two buttons will have the same behavior and there is no need to keep both of them. 

Maybe we could just remove `Continue` to make things simpler?

**Comment by ConnorCartwright** [comment]
Hey Shawn @dinever 

I think you're right - removing the `Continue` button would make things simpler.

I thought the `Save Changes` button didn't lead to allowing you to invite admins, but seeing as both buttons lead to the same page it would make more sense to remove 'Continue' (unless we plan on having more setup options later on).

**Comment by johndbritton** [comment]
Yeah, I don't know why we have two buttons there.

@tarebyte any insight into this? I think `Continue` makes the most sense as the button to keep since this is part of the creation flow. Maybe this is caused because we want to display `Save changes` on the edit page and `Continue` in the creation flow.

File: app/assets/javascripts/update-organization.coffee [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$(".update_organization").on 'click', ->
+  $('form.edit_organization').submit();
```
Comment (tarebyte) [inline app/assets/javascripts/update-organization.coffee]: Can you add a newline here? I need to add a JS linter to catch this kind of stuff.

File: app/views/organizations/setup.html.erb [hunk 2]
```diff
@@ -17,8 +17,8 @@
       </div>
 
       <div class="form-actions">
-        <%= f.submit 'Save changes', class: 'btn btn-green' %>
-        <%= link_to 'Continue', invite_organization_path(@organization), class: 'btn', role: 'button' %>
+        <%= submit_tag 'Save changes', :type => 'button', class: 'btn btn-green update_organization' %>
```
Comment (tarebyte) [inline app/views/organizations/setup.html.erb]: Can we put this back to using `f.submit` since this button is part of the form?

**Comment by tarebyte** [comment]
> Maybe this is caused because we want to display `Save changes` on the edit page and `Continue` in the creation flow.

I'm pretty sure that was why @ConnorCartwright let's make continue the only button on that page, and have it perform the save action.

**Comment by ConnorCartwright** [comment]
@tarebyte I've removed the .coffeescript file and made the only button `Continue` :thumbsup:

File: app/assets/javascripts/update-organization.coffee (hunk 1 above)
Comment (dinever) [inline app/assets/javascripts/update-organization.coffee]: @tarebyte I'm adding the CoffeeScript linter right now.

**Comment by tarebyte** [comment]
Looks good, thanks @ConnorCartwright!

**Pull request merged**